src/components/BnListbox/BnListbox.spec.ts

```diff
@@ -119,7 +119,6 @@ describe('BnListbox', () => {
     wrapper.findComponent(ListboxOption).trigger('click');
     wrapper.find('form').trigger('submit');
 
-    await flushPromises();
```

Comment:
How do you know when to use this helper?

Reply:
I don't remember what made me use it in the first place, but the documentation mentions that it's for promises that are not handled directly by Vue (like http requests)

https://test-utils.vuejs.org/guide/advanced/async-suspense.html#resolving-other-asynchronous-behavior